src/libxrpl/protocol/STTx.cpp

```diff
@@ -256,6 +258,32 @@ STTx::checkSign(
     return Unexpected("Internal signature check failure.");
 }
 
+Expected<void, std::string>
+STTx::checkBatchSign(
+    RequireFullyCanonicalSig requireCanonicalSig,
+    Rules const& rules) const
+{
+    try
```

Comment:
Should this have an assert confirming that this is a `Batch` transaction?

Reply:
fixed: https://github.com/XRPLF/rippled/pull/5060/commits/c5db6d977d5ef21a2cfbfa5321c70e4266fa14a9